samcli/commands/local/lib/swagger/integration_uri.py

```diff
@@ -117,14 +124,17 @@ def _get_function_arn(uri_data):
             return None
 
         # uri_data is a string.
-        # Let's check if it is actually a Lambda Integration URI and if so, extract the Function ARN
-        matches = re.match(LambdaUri._REGEX_GET_FUNCTION_ARN, uri_data)
-        if not matches or not matches.groups():
-            LOG.debug("Ignoring Integration URI because it is not a Lambda Function integration: %s", uri_data)
-            return None
+        # Extract the first Function ARN
+        matches = re.search(LambdaUri._REGEX_GET_FUNCTION_ARN, uri_data)
```

Comment:
Nit: we could go through series of regex handlers which returns the first match, however not blocking on this as this is readable.